README.md

```diff
@@ -396,6 +396,7 @@ are not maintained by the `tokio` project. These include:
 - [`test-env-log`] takes care of initializing `tracing` for tests, based on
   environment variables with an `env_logger` compatible syntax.
 - [`tracing-unwrap`] provides convenience methods to report failed unwraps on `Result` or `Option` types to a `Subscriber`.
+- [`diesel-tracing`] provides integration with diesel database connections.
```

Comment:
Nit, take it or leave it: it might be nice to link to Diesel here?

Reply:
Ah good point, have updated the commit to include a link!